server/controllers/shared/risksAndNeedsController.test.ts

```diff
@@ -174,6 +174,8 @@ describe('RisksAndNeedsController', () => {
         ...sharedPageData,
         alcoholMisuseSummaryListRows,
         hasData: true,
+        hideTitleServiceName: true,
```

Comment:
All the instances of `hideTitleServiceName` and `pageTitleOverride` in this file could get added to `sharedPageData` on line 122 to avoid the repetition.